salt/metalk8s/container-engine/containerd/installed.sls

```diff
@@ -8,24 +8,30 @@
 include:
   - metalk8s.repo
 
+{%- if grains['os_family'] == 'RedHat' %}
 Install container-selinux:
   {{ pkg_installed('container-selinux') }}
     - require:
       - test: Repositories configured
+{%- endif %}
```

Comment:
Suggest you use `require_in` instead of adding extra `if` every where. 
https://docs.saltstack.com/en/latest/ref/states/requisites.html#the-in-versions-of-requisites

Something like
```
Install container-selinux:
￼	  {{ pkg_installed('container-selinux') }}
￼	    - require:
￼	      - test: Repositories configured
            - require_in:
              - pkg: Install runc
```

For me the require in `Install containerd` for `container-selinux` is useless as `Install containerd` already require `runc` but it's not really part of your PR.

Reply:
Indeed, I would like to open a new PR about the different packets required for the node installation. But I keep your suggestion in mind